src/OrleansGCPUtils/Providers/Streams/PubSub/IPubSubDataAdapter.cs

```diff
@@ -7,12 +7,12 @@
 using System.Collections.Generic;
 using System.Linq;
 
-namespace Orleans.Providers.Streams
+namespace Orleans.Providers.GCP.Streams.PubSub
 {
     /// <summary>
     /// Converts event data to and from cloud queue message
     /// </summary>
-    public interface IGooglePubSubDataAdapter
```

Comment:
Let's either call it GCPPubSub, or if it already lives under GCP namespace, just leave as PubSub. No need to duplicate namespace in the name.

Both of the above 2 sound good to me.